extensions/blocks/eventbrite/eventbrite.php

```diff
@@ -63,25 +129,84 @@ function render_block( $attr, $content ) {
 				iframeContainerId: '" . esc_js( $widget_id ) . "',
 			} );"
 		);
+	}
+
+	return sprintf(
+		'<div id="%1$s" class="%2$s">%3$s</div>',
+		esc_attr( $widget_id ),
+		esc_attr( $classes ),
+		$embed
+	);
+}
+
+/**
+ * Render block with modal style.
+ *
+ * @param array  $attr    Eventbrite block attributes.
+ * @param string $content Rendered embed element (without scripts) from the block editor.
+ * @return string Rendered block.
+ */
+function render_modal_block( $attr, $content ) {
+	$widget_id = wp_unique_id( 'eventbrite-widget-' );
 
-		// $content contains a fallback link to the event that's saved in the post_content.
-		// Append a div that will hold the iframe embed created by the Eventbrite widget.js.
-		$classes = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attr );
+	$is_amp = Jetpack_AMP_Support::is_amp_request();
 
-		$content .= sprintf(
-			'<div id="%1$s" class="%2$s"></div>',
-			esc_attr( $widget_id ),
-			esc_attr( $classes )
+	if ( $is_amp ) {
+		$lightbox_id = "{$widget_id}-lightbox";
+
+		// Add CSS to for lightbox.
+		Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
```

Comment:
Don't we need the CSS for the non-AMP modal as well?

Reply:
Not as far as I can tell, no. The CSS gets injected into the page by the Eventbrite widget on the non-AMP version.